pkg/minishift/registration/utils.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright (C) 2016 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package registration
+
+import (
+	"fmt"
+	"github.com/docker/machine/libmachine/provision"
+	"github.com/docker/machine/libmachine/host"
+	"github.com/docker/machine/libmachine/drivers"
+)
+
+
+type RegistrationParametersStruct struct {
+	Username      string
+	Password      string
+	Proxy         string
+	Proxyuser     string
+	Proxypassword string
+}
+
+// Register host VM
+func RegisterHostVM(host *host.Host, param *RegistrationParametersStruct) error {
```

Comment:
This is one more level of abstraction which we can directly use for start command without putting lot of info in the start.go file.